web/src/components/base/Header.js

```diff
@@ -192,22 +173,17 @@ function Header({ setSearchResults, setLoading }) {
       return (
         <>
           <div className="helloUsername">Hello {username}!</div>
-          <button
-            onClick={navigateToGuestDashboard}
-            className="dropdownLoginButton"
-          >
+          <button onClick={navigateToGuestDashboard} className="dropdownLoginButton">
             Profile
           </button>
           <button onClick={navigateToMessages} className="dropdownLoginButton">
             Messages
           </button>
-          <button onClick={navigateToPayments} className="dropdownLoginButton">
+
+          <button onClick={() => navigate("/guestdashboard/payments")} className="dropdownLoginButton">
             Payments
           </button>
-          {/* <button onClick={navigateToReviews} className="dropdownLoginButton">
-            Reviews
-          </button> */}
-          <button onClick={navigateToSettings} className="dropdownLoginButton">
+          <button onClick={() => navigate("/guestdashboard/settings")} className="dropdownLoginButton">
```

Comment:
Hetzelfde geldt voor dit
```suggestion
          <button onClick={() => navigate("/guestdashboard/settings")} className="dropdownLoginButton">
```